halo2_gadgets/src/sha256/table16/compression.rs

```diff
@@ -922,7 +922,7 @@ impl CompressionConfig {
         layouter: &mut impl Layouter<pallas::Base>,
         state: State,
     ) -> Result<[BlockWord; DIGEST_SIZE], Error> {
-        let mut digest = [BlockWord(Value::known(0)); DIGEST_SIZE];
+        let mut digest = [BlockWord(IntegerValue::ZERO); DIGEST_SIZE];
```

Comment:
Here the result is actually longer, as for primitive integers the type can often be elided.